src/renderer/store/InstanceData.ts

```diff
@@ -1,7 +1,10 @@
+import { Version } from "@xmcl/core";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Version' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5163001)